docs/workflow-and-job.md

```diff
@@ -10,151 +10,222 @@ It can execute anything that can be executed from the Python code, for example:
 * BigQuery query
 * Any Python code.
 
-The simplest workflow you can create, looks like this:
+The simplest workflow you can create looks like this:
 
+[`./docs_examples/workflow_and_job/simple_workflow_and_job.py`](./docs_examples/workflow_and_job/simple_workflow_and_job.py)
 ```python
 from bigflow.workflow import Workflow
 
-class HelloWorldJob:
+class SimpleJob:
     def __init__(self):
-        self.id = docs
+        self.id = 'simple_job'
 
     def run(self, runtime):
-        print(f'Hello world at {runtime}!')
+        print(f'Running a simple job')
 
-hello_world_workflow = Workflow(workflow_id='hello_world_workflow', definition=[HelloWorldJob()])
+simple_workflow = Workflow(workflow_id='simple_workflow', definition=[SimpleJob()])
 ```
 
-Now, you can run the `hello_world_workflow`:
+You can run the workflow and job within a module (but we recommend using the BiggerQuery CLI):
 
-```shell script
-bgf run --workflow hello_world_workflow
+[`./docs_examples/workflow_and_job/run_in_module.py`](./docs_examples/workflow_and_job/run_in_module.py)
+```python
+simple_workflow.run()
+simple_workflow.run_job('simple_job')
 ```
 
-Or a single job:
+Output:
 
-```shell script
-bgf run --job hello_world_workflow.hello_world
+```text
+Running a simple job
+Running a simple job
 ```
 
-You can also run the workflow and job within the module:
-
-```python
-hello_world_workflow.run()
-hello_world_workflow.run_job('hello_world')
-```
+## Job
 
-Finally, you can build and deploy created workflow to Cloud Composer:
+A job is just an object that has a unique `id` and implements the `run` method.
 
-```shell script
-bgf build;bgf deploy
-```
+The `id` parameter is a string that should be a valid Python variable name. For example - `'my_example_job'`, `'MY_EXAMPLE_JOB'`, `'job1234'`.
 
-### Job
+The run method takes a single argument - `runtime`. The `runtime` parameter is a date-time in the form of a string. 
+You can find more information about `runtime` and scheduling [in the chapter below](#workflow-scheduling-options).
 
-Job is just an object that has a unique `id` and implements the `run` method.
+There are 2 additional parameters, that a job can supply - `retry_count` and `retry_pause_sec`. The `retry_count` parameter
+determines how many times a job will be retried (in case of a failure). The `retry_pause_sec` says how long the pause between retries should be.
```

Comment:
`retry_pause_sec` or The `retry_pause_sec` parameter

Reply:
done